apps/test/unit/templates/studioHomepages/SectionsTableTest.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import i18n from '@cdo/locale';
+import {joinedSections} from './homepagesTestData';
+import SectionsTable from '@cdo/apps/templates/studioHomepages/SectionsTable';
+
+describe('SectionsTable', () => {
+
+  it('shows column headers for students with teacher-managed accounts', () => {
+    const wrapper = shallow(
+      <SectionsTable
+        sections={joinedSections}
+        isRtl={false}
+        isTeacher={false}
+        canLeave={false}
+      />
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.section()}
+        </div>
+      </td>
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.course()}
+        </div>
+      </td>
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.teacher()}
+        </div>
+      </td>
+    );
+    expect(wrapper).to.containMatchingElement(
+      <td>
+        <div>
+          {i18n.sectionCode()}
+        </div>
+      </td>
+    );
+    // Students with teacher-managed accounts should not see a button to leave the section.
+    expect(wrapper.find('Button').exists()).to.be.false;
```

Comment:
Your test case name makes it sound like we're checking column headers, but the Button is always rendered in the table body.  And the comment here is sort of like a test case name in its own right. I would break this out into its own test case.